lib/model/pom.xml

```diff
@@ -56,4 +56,46 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <!-- override plugin to include the jandex resource -->
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>${maven-bundle-plugin.version}</version>
+        <inherited>true</inherited>
+        <extensions>true</extensions>
+        <configuration>
+          <excludeDependencies>${osgi.exclude.dependencies}</excludeDependencies>
+          <instructions>
+            <Bundle-Name>${project.name}</Bundle-Name>
+            <Bundle-SymbolicName>${osgi.symbolic.name}</Bundle-SymbolicName>
+            <Bundle-DocURL>${project.url}</Bundle-DocURL>
+            <Export-Package>${osgi.export.pkg}</Export-Package>
+            <Import-Package>${osgi.import.pkg}</Import-Package>
+            <Export-Service>${osgi.export.service}</Export-Service>
+            <Implementation-Title>${project.description}</Implementation-Title>
+            <Implementation-Version>${project.version}</Implementation-Version>
+            <Embed-Dependency>${osgi.embed.dependency}</Embed-Dependency>
+            <Embed-Transitive>${osgi.embed.transitive}</Embed-Transitive>
+            <Include-Resource>${osgi.include.resource},META-INF/jandex.idx=target/classes/META-INF/jandex.idx
```

Comment:
Couldn't it be like `<osgi.include.resource>${osgi.include.resource},META-INF/jandex.idx=target/classes/META-INF/jandex.idx</osgi.include.resource>` under properties?

Reply:
Or, in parent/pom.xml
```xml
<properties>
  <osgi.include.resource.atlasmap>
    ${osgi.include.resource}
  </osgi.include.resource.atlasmap>
</properties>
...
<Include-Resource>${osgi.include.resource.atlasmap}</Include-Resource>
```
then in children which needs to override
```xml
<properties>
  <osgi.include.resource.atlasmap>
    ${osgi.include.resource},META-INF/jandex.idx=target/classes/META-INF/jandex.idx
  </osgi.include.resource.atlasmap>
</properties>

Reply:
@igarashitm this sounds a good idea.